autoload/neomake/makers/ft/scala.vim

```diff
@@ -1,11 +1,23 @@
 " vim: ts=4 sw=4 et
 function! neomake#makers#ft#scala#EnabledMakers()
-    let makers = ['scalac']
-    if neomake#utils#Exists('scalastyle')
+    let makers = get(g:, 'neomake_scala_fast', 0) ? ['fsc'] : ['scalac']
+    if neomake#utils#Exists('scalastyle') && get(g:, 'neomake_scala_style', 1)
```

Comment:
On second thought, I don't think it's the right thing to add a configuration option for every checker that's not enabled by default. Perhaps it's a good time to set up a wiki that explains how to get set up for each file type so this info will be googleable.